src/DataTable.php

```diff
@@ -392,7 +392,15 @@ public function getColLabels(array $cols = null): array
             }
         });
 
-        return $colLabels;
+        return array_merge(
+            $colLabels,
+            [
```

Comment:
would recommend swapping the arrays in array_merge. Otherwise one could never overwrite these.